src/atoms/TextInput/TextInput.native.js

```diff
@@ -448,6 +473,10 @@ TextInput.defaultProps = {
   testID: 'ds-text-input',
   labelPosition: 'top',
   width: 'medium',
+  secureTextEntry: false,
```

Comment:
can we accept a prop called `type` and then map internally to secureTextEntry? which then remains close to web as well?
so it'll be `type="text"` or `type="password"`